File: CS50x/pset6/mario.py
import cs50

# Asks for user's input and validate results. Height should be positive and no greater than 23.
while True:
    print("Height:", end="")
    a = cs50.get_int()
    if a >= 0 and a <= 23:
        break
# Iterate over a (height) to create rows and prints blocks and spaces as needed.    
for i in range(a):
    print(" " * (a - i - 1), end="")
    print("#" * (i + 2), end="")
    print()
    

    
    

File: CS50x/pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int a;
    int b = 0;
    
    do
    {
        printf("Height:");
        a = get_int();
    }
    while (a < 0 || a > 23);
    
    for (b = 0; b < a; b++)
    {
    int n = 0;
    int j = 0;
        for(n = 0; n < a - b - 1; n++)
        {
            printf(" ");
        }
   
        for(j = 0; j <= b + 1; j++)
        {
            printf("#");
        }
        
    printf("\n"); 
        
    }    
    

}
